### PR TITLE
Fix sync for qa (HPC-9459)

### PR DESCRIPTION
**Pull request by itkovian**
This requires https://github.com/hpcugent/vsc-config/pull/144 to be merged

### CODE REVIEW

File: bin/sync_slurm_acct.py [hunk 1]
```diff
@@ -75,6 +75,12 @@ def main():
             None,
         ),
         'start_timestamp': ('Timestamp to start the sync from', str, 'store', None),
+        'cluster_classes': (
+            'Classes of clusters that should be synced, comma-separated',
```
Comment (stdweird) [inline bin/sync_slurm_acct.py:79]: generaloption has a strlist type

File: bin/sync_slurm_acct.py [hunk 2]
```diff
@@ -99,7 +105,7 @@ def main():
         if opts.options.clusters is not None:
             clusters = opts.options.clusters.split(",")
         else:
-            clusters = VSC_SLURM_SYNC_CLUSTERS[host_institute]
+            clusters = [cs for p in opts.options.cluster_classes.split(',') for cs in  VSC_SLURM_SYNC_CLUSTERS[host_institute]]
```
Comment (stdweird) [inline bin/sync_slurm_acct.py:108]: i guess a `[p]` is missing somewhere at the end?

File: bin/sync_slurm_acct.py (hunk 1 above)
Comment (itkovian) [reply bin/sync_slurm_acct.py:79]: right, I missed that.

File: bin/sync_slurm_acct.py [hunk 3]
```diff
@@ -75,6 +75,12 @@ def main():
             None,
         ),
         'start_timestamp': ('Timestamp to start the sync from', str, 'store', None),
```
Comment (stdweird) [inline bin/sync_slurm_acct.py:77]: a few lines higher, also user strlist (and remov ethe split below); but also adapt the help to the new variables and explain what really is the default now

Comment (itkovian) [reply bin/sync_slurm_acct.py:77]: Change creep :p

File: bin/sync_slurm_acct.py [hunk 4]
```diff
@@ -96,10 +102,13 @@ def main():
...
 
-        if opts.options.clusters is not None:
-            clusters = opts.options.clusters.split(",")
+        if opts.options.clusters:
+            clusters = opts.options.clusters
         else:
-            clusters = VSC_SLURM_SYNC_CLUSTERS[host_institute]
+            clusters = [cs
+                for p in opts.options.cluster_classes
+                for cs in  VSC_SLURM_CLUSTERS[host_institute][p]
```
Comment (stdweird) [inline bin/sync_slurm_acct.py:110]: double space after `in`

**Pull request merged**